### PR TITLE
Initial setup of a new el8 based gwms 3.9.4 image

### PR DESCRIPTION
**Pull request by rynge**

### CODE REVIEW

**Comment by rynge** [comment]
My preference would be to only build this for 3.6

File: opensciencegrid/vo-frontend/.github/workflows/build-sw-container.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: Build and Push Docker image
```
Comment (brianhlin) [inline opensciencegrid/vo-frontend/.github/workflows/build-sw-container.yml:1]: You can drop this file completely, we use what's in `.github` of the root of this dir now

**Review by brianhlin — APPROVED** [review]
LGTM with one minor comment

File: opensciencegrid/vo-frontend/.github/workflows/build-sw-container.yml (hunk 1 above)
Comment (rynge) [reply opensciencegrid/vo-frontend/.github/workflows/build-sw-container.yml]: Done.

**Pull request merged**